### DISCUSSION TITLE
Hide the AWS ID and Azure Certificate token when they are not configured

### DISCUSSION THREAD

**Discussion by thinkst-pieter** [thinkst/canarytokens#270] [Feature Requests]
The AWS ID and Azure Certificate depend on external infrastructure to operate. 

We should hide the option to create these tokens when the configuration to point to the external infrastructure is not present.
Additionally we need to update the documentation to indicate the requirements for these tokens more clearly

**Comment by jayjb** [comment]
@wleightond: this is sorted right?

**Comment by wleightond** [comment]
Yup! This one is closed by #272. 

I'm assuming the docs component should be over in [`thinkst/canarytokens-docker`](https://github.com/thinkst/canarytokens-docker/)
